src/index.html

```diff
@@ -17,35 +17,35 @@
   </head>
   <body>
     <header class="header">
-      <a class="header__logo" href="">
+      <a class="header__logo" href="@">
         <img src="./images/logo.png" alt="Moyo logo">
       </a>
 
       <nav class="nav">
         <ul class="nav__list">
           <li class="nav__item nav__item--is-active">
-            <a href="" class="nav__link">Apple</a>
+            <a href="@" class="nav__link">Apple</a>
```

Comment:
add the proper links in all cases
```suggestion
            <a href="#apple" class="nav__link">Apple</a>
```